gammapy/datasets/simulate.py

```diff
@@ -88,6 +88,8 @@ def sample_sources(self, dataset):
             table = self._sample_coord_time(npred, temporal_model, dataset.gti)
             if len(table) > 0:
                 table["MC_ID"] = idx + 1
+                table.meta["MID{:05d}".format(idx + 1)] = idx + 1
+                table.meta["MMN{:05d}".format(idx + 1)] = dataset.models[idx].name
```

Comment:
You can directly use:
```suggestion
                table.meta["MMN{:05d}".format(idx + 1)] = evaluator.model.name
```

Reply:
Thanks @adonath . I've implemented all suggestions.